### ISSUE TITLE
HMG

### ISSUE THREAD

**Issue by Afavertov** [studio-minus/ppg-bugs-and-requests#573]
Machinegun like M2 or DSHK, that much powerful and accurate then LMG, but have less firerate.

LMG is weak, but MG is literally autocannon, that can pierce buildings, so this will something between them.

**Comment by mestiez** [comment]
Isn't the minigun technically a heavy machine gun? I'm guessing it fires way too fast for your purposes though

**Comment by Afavertov** [comment]
Tank with minigun as secondary weapon look strange.

**Comment by satsubatsu347** [comment]
> Isn't the minigun technically a heavy machine gun? I'm guessing it fires way too fast for your purposes though

No. The M134 is a "rotary medium machine gun," believe it or not. "Minigun" comes from re-specing Vulkan Autocannons to 7.62x51mm NATO (.308 equivalent). An M2 is a heavy machine gun and are .50 caliber (aka 12.7×99mm NATO). Think door gunners, machine gun nests, and deck guns. Although a famed sniper did modify one to be used as a sniper rifle with great success.

**Comment by Afavertov** [comment]
I still want M2 or KPV in game. Using sniper with metronome isnt actually fun
